packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -303,40 +335,87 @@ export const getStyles = memoizeFunction(
       ],
 
       rootHovered: {

```

Comment:
Probably revert the changes to the rootHovered section (looks like those changes all pertain to the previous approach of using :after for all borders)